xonsh/jobs.py

```diff
@@ -447,53 +453,50 @@ def bg(args, stdin=None):
         return res
 
 
-def disown(args, stdin=None):
-    """
-    xonsh command: disown
+# todo: add tests for completer
```

Comment:
Let me know when this is done and I'll merge

Reply:
added tests